frontend/packages/shared/src/api/mutations.ts

```diff
@@ -46,9 +50,10 @@ import { buildQueryParams } from 'app-shared/utils/urlUtils';
 import type { JsonSchema } from 'app-shared/types/JsonSchema';
 import { CreateDatamodelPayload } from 'app-shared/types/api/CreateDatamodelPayload';
 import type { Policy } from '@altinn/policy-editor';
-import type { NewResource, Resource } from 'app-shared/types/ResourceAdm';
+import type { NewResource, PartyList, Resource } from 'app-shared/types/ResourceAdm';
 import { ApplicationMetadata } from 'app-shared/types/ApplicationMetadata';
 import { AppConfig } from 'app-shared/types/AppConfig';
+import { JsonPatch } from 'resourceadm/utils/jsonPatchUtils/jsonPatchUtils';
```

Comment:
A file in the ```shared``` folder should not import from a space-specific file. I would suggest to move the ```jsonPatchUtils``` folder to ```shared``` if it is needed in shared.